src/query/formats/src/output_format/json.rs

```diff
@@ -119,6 +119,10 @@ fn scalar_to_json(s: ScalarRef<'_>, format: &FormatSettings) -> JsonValue {
                 .collect();
             JsonValue::Object(vals)
         }
+        ScalarRef::Bitmap(b) => {
+            let b = jsonb::from_slice(b).unwrap();
```

Comment:
`Bitmap` is not a `jsonb` value, need to convert the binary `RoaringBitmap` to a JSON array of integers.